js/cli_utils.ts

```diff
@@ -0,0 +1,93 @@
+import { TextEncoder } from "./text_encoding";
+
+const encoder = new TextEncoder();
+
+const tableChars = {
+  middleMiddle: "─",
+  rowMiddle: "┼",
+  topRight: "┐",
+  topLeft: "┌",
+  leftMiddle: "├",
+  topMiddle: "┬",
+  bottomRight: "┘",
+  bottomLeft: "└",
+  bottomMiddle: "┴",
+  rightMiddle: "┤",
+  left: "│ ",
+  right: " │",
+  middle: " │ "
+};
+
+const colorRegExp = /\u001b\[\d\d?m/g;
+
+function removeColors(str: string) {
+  return str.replace(colorRegExp, "");
+}
+
+const countBytes = (str: string) => {
+  const normalized = removeColors(String(str)).normalize("NFC");
+
+  return encoder.encode(normalized).byteLength;
+};
+
+const renderRow = (row: string[], columnWidths: number[]) => {
+  let out = tableChars.left;
+  for (let i = 0; i < row.length; i++) {
+    const cell = row[i];
+    const len = countBytes(cell);
+    const needed = (columnWidths[i] - len) / 2;
+    // round(needed) + ceil(needed) will always add up to the amount
+    // of spaces we need while also left justifying the output.
+    out += `${" ".repeat(needed)}${cell}${" ".repeat(Math.ceil(needed))}`;
+    if (i !== row.length - 1) {
+      out += tableChars.middle;
+    }
+  }
+  out += tableChars.right;
+  return out;
+};
+
+export const cliTable = (head: string[], columns: string[][]) => {
```

Comment:
Use `export function cliTable(...` 

https://github.com/denoland/deno_std#top-level-functions-should-not-use-arrow-syntax

Also add return type.